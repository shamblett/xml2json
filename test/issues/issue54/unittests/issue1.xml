<?xml version="1.0" encoding="utf-8"?>
<!--
 Test Case submitted in issue 1:
 http://code.google.com/p/xml2json-xslt/issues/detail?id=1
 -->
<productRating>
   <rating>
     <ratingCode>PG-13</ratingCode>
   </rating>
   <explaination/>
</productRating>
