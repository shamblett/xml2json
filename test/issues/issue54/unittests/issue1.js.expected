/*
 Test Case submitted in issue 1:
 http://code.google.com/p/xml2json-xslt/issues/detail?id=1
 */{
    rating:{
      ratingCode:"PG-13"},
    explaination:null}
